learn/wme-backup-restore/wme-backup-restore-introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "WME-Backup-Restore"
+id: "wme-backup-restore-introduction"
+sidebar_label: "Introduction"
+---
+---
+## Introduction
+
+WaveMaker uses separate dedicated directory for storing data, which is easy to take backup and restore
+
```

Comment:
Platform Direcotory Structure Image
StudioWorkspace Direcotory Structure Image (Dont use External Instance. Use StudioWorkspace Instance or AppDeployment Instance).. these images should have all directories used by Platform, irresepective of state
Explain brifly about above  images. Explain about how do they can allocate disks/volumes for each directory.
Current image are not looking good. remove italic. little low color. work with swetha if needed

Reply:
What to be Backup
When to Restore
write only bullet points

Reply:
Prequisities for Backup and Restore.
What happens if we take backup of Platform and StudioWorkspace instance in different times etc
Explain about two ways, Directories and Disk/Volumes